test-apps/crypto-trading/aggregates/src/main/java/org/elasticsoftware/cryptotrading/aggregates/orders/OrderProcess.java

```diff
@@ -30,26 +31,27 @@
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
 @JsonSubTypes({
-        @JsonSubTypes.Type(value = BuyOrderProcess.class, name = "BUY")
+        @JsonSubTypes.Type(value = BuyOrderProcess.class, name = "BUY"),
+        @JsonSubTypes.Type(value = SellOrderProcess.class, name = "SELL")
 })
-public sealed interface OrderProcess extends AkcesProcess permits BuyOrderProcess {
+public sealed interface OrderProcess extends AkcesProcess permits BuyOrderProcess, SellOrderProcess {
     String orderId();
 
     CryptoMarket market();
 
     BigDecimal size();
 
-    BigDecimal amount();
+    BigDecimal quantity();
```

Comment:
keep using amount. For the Sell order process use size (which is identical to quantity)